### DISCUSSION TITLE
Anyone using this for datacenters?

### DISCUSSION THREAD

**Discussion by ryanmerolle** [inventree/InvenTree#3176] [General]
I use [NetBox ](https://github.com/netbox-community/netbox )quite extensively for datacenter management and I provide [Snipe-IT](https://github.com/snipe/snipe-it) to our desktop support folk.   I found Snipe-It was too user focused to provide complimentary spares/parts inventory tooling to NetBox since.

Anyone using this tool with NetBox?

It could be interesting to either create a NetBox plugin and/or a InvenTree plugin to link the two.

**Comment by matmair** [comment]
Hi there @ryanmerolle ! I am running netbox and snipe-it myself at work and my recommendation would be to link those two together instead of rolling out InvenTree.
We are defacto a PLM / for manufacturing and miss several things a bigger IT needs. Especially:
- LDAP / AD sync (you can log in via LDAP and create users at that point)
- Multiple tokens per user
- multi-tenant / multi-company
- Acceptance / Request flows
- Signatures

You could roll that up but I think it is faster to link snipe-it with netbox.

**Reply by ryanmerolle** [reply]
I find snipe-it is very user focused and wants you to checkout assets per user which is not ideal for datacenter work.

**Reply by matmair** [reply]
You are free to use what you want. I think InvenTree will have shortcomings that will cause problems in any org that follows industry best practices like ISO9001 or ISO27001 .

**Comment by ryanmerolle** [comment]
Oh sorry, hopefully I did not come off negatively.  I was sharing for the discussion and I’m very appreciative that you are sharing your experience!
